respondd.py

```diff
@@ -125,22 +93,12 @@ def join_group(mcast_group, if_index=0):
             mreq
         )
 
-    # Extract multicast interfaces from commandline parameters
-    mcast_iface_groups = { }
-    for ifspec in args.mcast_ifaces:
-        iface, *groups = reversed(ifspec.split('%'))
-        # Populate with default link and site mcast groups if entry not yet created
-        if not iface in mcast_iface_groups:
-            mcast_iface_groups[iface] = [ group for group in [ args.link_group, args.site_group ] if len(group) > 0 ]
-        # Append group specified on commndline
-        mcast_iface_groups[iface] += groups
-
     for (if_index, if_name) in socket.if_nameindex():
         # Check if daemon should listen on interface
-        if if_name in mcast_iface_groups:
-            groups = mcast_iface_groups[if_name]
+        if if_name in DomainRegistry.get_instance().get_interfaces():
```

Comment:
This condition only returns true for batman interfaces, which is why multicast binds on other interfaces are missing right now,.

Reply:
Huh, why would that be? It should return the batman interface plus all interfaces specified in the domains 'Interfaces' config section. I've added a commit to state more cleary what format the interfaces are supposed to be specified in

Reply:
Is the config I posted wrong? https://gist.github.com/mweinelt/696d5b33c08944a1f65ccb439b346cf2

Reply:
Small typo, should be `Interfaces` not `Interface` in the domain config

Reply:
Good catch!